phoenix-hive/src/main/java/org/apache/phoenix/hive/PhoenixSerializer.java

```diff
@@ -64,12 +64,40 @@
 
     public PhoenixSerializer(Configuration config, Properties tbl) throws SerDeException {
         String mapping = tbl.getProperty(PhoenixStorageHandlerConstants.PHOENIX_COLUMN_MAPPING, null);
-        if(mapping!=null ) {
+        if(mapping != null) {
             config.set(PhoenixStorageHandlerConstants.PHOENIX_COLUMN_MAPPING, mapping);
         }
+
+        String tableName = tbl.getProperty(PhoenixStorageHandlerConstants.PHOENIX_TABLE_NAME);
+        config.set(PhoenixStorageHandlerConstants.PHOENIX_TABLE_NAME, tableName);
+
+        config.set(PhoenixStorageHandlerConstants.PHOENIX_ROWKEYS,
+          tbl.getProperty(PhoenixStorageHandlerConstants.PHOENIX_ROWKEYS));
+
+        String quorum = config.get(PhoenixStorageHandlerConstants.ZOOKEEPER_QUORUM);
+        if (quorum == null) {
```

Comment:
Is it possible that `quorum` is ever non-null? A log message if the unexpected happens would be helpful.

Reply:
Thank you for this comment. It looks like `quorum` from `config` can be null. In this case, the quorum value from table property `tbl` will be used.

The same logic here:
https://github.com/apache/phoenix-connectors/blob/0d50ca517c41071413743dafed519d96b58baf69/phoenix-hive/src/main/java/org/apache/phoenix/hive/util/PhoenixConnectionUtil.java#L47-L61

Reply:
Hehe, thanks Toshi. I wish we understood better how Hive would call this code as it would help us write some more terse code :)